histomicstk/cli/ColorDeconvolution/ColorDeconvolution.py

```diff
@@ -89,9 +84,17 @@ def main(args):
                     'yoffset': region.get('top', 0),
                 },
             }],
-        }]
+        },
```

Comment:
Rather than create another annotation, this should be added to the current annotations.